scripts/unix_share_build.sh

```diff
@@ -66,13 +95,12 @@ function build_moab() {
     cd bld
     cmake ../moab -DENABLE_HDF5=ON \
             -DHDF5_ROOT=$HDF5_PATH \
-            -DBUILD_SHARED_LIBS=ON \
+            -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS} \
             -DENABLE_BLASLAPACK=OFF \
             -DENABLE_FORTRAN=OFF \
             $CMAKE_ADDITIONAL_FLAGS \
             -DCMAKE_INSTALL_PREFIX=${PLUGIN_ABS_PATH}/moab
-
-    make -j$PROC
```

Comment:
I'm curious about this. Why did it need to change? Should we select only a few processes instead?

Reply:
I removed it because we are not here for speed, and this something causes problems with `gcc`....